From cfa9537e3a35ade9d2bba91143559d4e5a22397e Mon Sep 17 00:00:00 2001
From: Manivannan Sadhasivam <manivannan.sadhasivam@linaro.org>
Date: Tue, 4 Dec 2018 18:33:26 +0530
Subject: [PATCH 40/45] mmc: dw_mmc: Add SD/MMC support for HI3670 SoC

HI3670 SoC is architecturally same as HI3660 SoC and the same Designware
Mobile Storage Host controller IP block is reused. Hence, add the HI3670
compatible to K3 driver reusing the HI3660 driver.

Signed-off-by: Manivannan Sadhasivam <manivannan.sadhasivam@linaro.org>
---
 drivers/mmc/host/dw_mmc-k3.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mmc/host/dw_mmc-k3.c b/drivers/mmc/host/dw_mmc-k3.c
index 89cdb3d533bb..670b1ea11ed1 100644
--- a/drivers/mmc/host/dw_mmc-k3.c
+++ b/drivers/mmc/host/dw_mmc-k3.c
@@ -447,6 +447,7 @@ static const struct dw_mci_drv_data hi3660_data = {
 
 static const struct of_device_id dw_mci_k3_match[] = {
 	{ .compatible = "hisilicon,hi3660-dw-mshc", .data = &hi3660_data, },
+	{ .compatible = "hisilicon,hi3670-dw-mshc", .data = &hi3660_data, },
 	{ .compatible = "hisilicon,hi4511-dw-mshc", .data = &k3_drv_data, },
 	{ .compatible = "hisilicon,hi6220-dw-mshc", .data = &hi6220_data, },
 	{},
-- 
2.15.1

